src/classes/OPP_OpportunityNaming.cls

```diff
@@ -306,11 +278,28 @@ public with sharing class OPP_OpportunityNaming implements OPP_INaming {
         for (string field : getConFields(fields)) {
             result += field + ', ';
         }
-        result = result.substring(0, result.length()-2) + ' FROM OpportunityContactRole';
+        result = result.substring(0, result.length()-2) + ' FROM OpportunityContactRoles';
```

Comment:
This was a query and is now a subquery, the subquery relationship name has an s at the end.